src/messages/pt/forms.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Message translations.
+ *
+ * This file is automatically generated by 'yii message/extract' command.
+ * It contains the localizable messages extracted from source code.
+ * You may modify this file by translating the extracted messages.
+ *
+ * Each array element represents the translation (value) of a message (key).
+ * If the value is empty, the message is considered as not translated.
+ * Messages that no longer need translation will have their translations
+ * enclosed between a pair of '@@' marks.
+ *
+ * Message string can be used with plural forms format. Check i18n section
+ * of the guide for details.
+ *
+ * NOTE: this file must be saved in UTF-8 encoding.
+ */
+return [
+    'Confirmation Step' => 'Mostrar resumo',
+    'Copy to Attribute' => 'Nome do atributo do destinatário',
+    'Customizing Text' => 'Personalizando o texto do e-mail',
+    'Form' => 'Formulário',
+    'Values' => 'Valores',
+    'checkboxes_values_help' => 'Se apenas um valor for especificado, uma única caixa de seleção será criada.',
+    'form_label_previewBackButtonLabel' => 'Rótulo do botão Voltar da visualização',
+    'form_label_previewButtonsTemplate' => 'Template for next and back button of preview',
+    'form_label_previewSubmitButtonLabel' => 'Modelo para o botão próximo e voltar da visualização',
+    'form_label_submitButtonLabel' => 'Rótulo do botão enviar do formulário',
+    'form_model_copy_to_attribute_hint' => 'Se o atributo for especificado e existir no formulário, o valor de seu campo será adicionado à lista de destinatários.',
+    'form_model_email_intro_hint' => 'O texto deste campo será usado como texto de introdução para o e-mail. Todas as variáveis ​​especificadas podem ser inseridas com {{attribute}}.',
+    'form_model_email_recipients_hint' => 'Se nenhum destinatário for fornecido, o envio do formulário não acionará um e-mail.',
+    'form_model_subject_hint' => 'Assunto do e-mail. Se não for especificado, o título do formulário será usado.',
+    'As Hidden Input Value' => '',
```

Comment:
There are many missing translations. Maybe you where taking a look at the en file, but as those strings are english, they don't need translation. Here in the german file you can see they are all translated.

https://github.com/luyadev/luya-module-forms/blob/master/src/messages/de/forms.php

Reply:
working on it 👍